src/microbe_stage/OrganelleTemplate.cs

```diff
@@ -13,26 +13,13 @@ public class OrganelleTemplate : IPositionedOrganelle, ICloneable
     [JsonProperty]
     public readonly OrganelleDefinition Definition;
 
-    public OrganelleTemplate(OrganelleDefinition definition, Hex location, int rotation, int numberOfTimesMoved = 0)
+    public OrganelleTemplate(OrganelleDefinition definition, Hex location, int rotation)
     {
         Definition = definition;
         Position = location;
         Orientation = rotation;
-        NumberOfTimesMoved = numberOfTimesMoved;
     }
 
-    /// <summary>
-    /// Used to flag whether this Organelle was placed during the current editor session.
-    /// </summary>
-    public bool PlacedThisSession { get; set; }
```

Comment:
Doesn't this flag need to be kept as there is a separate hex colour used for new hexes?